.gitignore

```diff
@@ -1,42 +1,21 @@
-*.rbc
-capybara-*.html
-.rspec
-/log
-/tmp
-/db/*.sqlite3
-/db/*.sqlite3-journal
-/public/system
```

Comment:
@domenic99 also I recommend to add string `*~` to `.gitignore` to ignore temporary files from editors.